apps/api/src/auth/auth.service.ts

```diff
@@ -225,13 +231,17 @@ export class AuthService {
     })
   }
 
-  private async createKeycloakUser(registerDto: RegisterDto, verifyEmail: boolean) {
+  private async createKeycloakUser(
+    registerDto: RegisterDto,
+    verifyEmail: boolean,
+    activeProfile = true,
+  ) {
     return await this.admin.users.create({
       username: registerDto.email,
       email: registerDto.email,
       firstName: registerDto.firstName,
       lastName: registerDto.lastName,
-      enabled: true,
+      enabled: activeProfile,
```

Comment:
The idea is to manually active corporate profiles after verification, to avoid situations such as Ivan Ivanov who has nothing to do with Google, to create corporate profile for Google and make donations using Google's name.
Whether this is actually needed I am unsure. 
How we will verify the corporate profile I'm also unsure.

Reply:
oh, yes! I remember now - thanks and this is indeed ok!